larky/src/main/java/com/verygood/security/larky/modules/types/structs/SimpleStruct.java

```diff
@@ -31,11 +35,28 @@ public static SimpleStruct mutable(Dict<String, Object> kwargs, StarlarkThread t
     return new MutableStruct(kwargs, thread);
   }
 
-  SimpleStruct(Map<String, Object> fields, StarlarkThread currentThread) {
+  public SimpleStruct(Map<String, Object> fields, StarlarkThread currentThread) {
```

Comment:
Does this need to be public? Can this be protected? I'd *really* like to make sure that we focus on creating factory.

Reply:
Sure, we can.